grpc-server/grpc_server.go

```diff
@@ -25,8 +21,8 @@ import (
 
 // Server is the gRPC server for tinkerbell
 type server struct {
-	cert []byte
-	modT time.Time
+	//cert []byte
+	//modT time.Time
```

Comment:
As far as I could tell from a quick glance, these didn't actually appear to be used anywhere outside of methods that were exposed and seemingly unused in `grpc-server/hardware.go`